crates/fuel-core/src/service/sub_services.rs

```diff
@@ -158,7 +159,8 @@ pub fn init_sub_services(
         config.chain_conf.gas_costs.clone(),
     )
     .data(database.clone());
-    let gql_database = Box::new(database.clone());
+
+    let query_data: Box<dyn QueryData> = Box::new(database.clone());
```

Comment:
Yeah, we definitely don't need a separate `query_data` type=) Because it is `Database`